api_server.py

```
@@ -75,107 +83,92 @@ def format_hours(minutes: Optional[int]) -> Optional[str]:
 def get_availability(entity_id: int, table: str, now: datetime) -> Dict:
     col = "crew_id" if table == "crew_availability" else "appliance_id"
-    with get_db() as conn:
-        curr = conn.execute(
-            f"SELECT end_time FROM {table} WHERE {col} = ? AND start_time <= ? AND end_time > ? LIMIT 1",
-            (entity_id, now, now),
-        ).fetchone()
-        if not curr:
-            return {"available": False, "duration": None, "end_time_display": None}
+    conn = get_db()
+    curr = conn.execute(
+        f"SELECT end_time FROM {table} WHERE {col} = ? AND start_time <= ? AND end_time > ? LIMIT 1",
```

Ruff (0.15.4)

[error] Possible SQL injection vector through string-based query construction

(S608)